staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/webhook.go

```diff
@@ -63,8 +64,8 @@ func StartConversionWebhookServer(handler http.Handler) (func(), *apiextensionsv
 	}
 
 	// StartTLS returns immediately, there is a small chance of a race to avoid.
-	if err := wait.PollImmediate(time.Millisecond*100, wait.ForeverTestTimeout, func() (bool, error) {
-		_, err := webhookServer.Client().Get(webhookServer.URL) // even a 404 is fine
+	if err := wait.PollUntilContextTimeout(context.Background(), time.Millisecond*100, wait.ForeverTestTimeout, true, func(ctx context.Context) (done bool, err error) {
+		_, err = webhookServer.Client().Get(webhookServer.URL) // even a 404 is fine
```

Comment:
this should rather stay `:=`. Otherwise we override the err outside which is asking for a race.

Reply:
Fixed it https://github.com/kubernetes/kubernetes/commit/ed7f7ce92b06b2452a3d879b4c70d34f5d31ff23